### ISSUE TITLE
Need shortcut to stop reading

### ISSUE THREAD

**Issue by danfrist** [pgmichael/wavenet-for-chrome#15]
The shortcut to start reading should also stop reading, or a second shortcut should be added. If you select a large section of text and start reading, you can't stop it!

**Comment by pgmichael** [comment]
Good point! In the meantime, if you have no text selected and press the start speaking shortcut it show stop the current sequence.

**Comment by danfrist** [comment]
That's helpful, thanks! 

Also, the shortcut to start reading doesn't work on PDFs in a chrome tab. I can, however, right-click and select Wavenet from the menu and it works. 

Also, the shortcut to start reading doesn't work on text inputs. I can, however, right-click and select Wavenet from the menu and it works.

**Comment by ischlag** [comment]
Hey @pgmichael found your plugin and had the same issue and wanted to add my thoughts. I'm using text to speech a lot at work and it really increases my productivity. I have a google api key and I've been using ReadAlout until it became unusable for me. Your plugin is my new goto but I find the usability is rather limited and a little clunky in my opinion.

I'd consider my self a power user when it comes to text to speech and I strongly agree with @danfrist that those are essential use cases for anyone that uses the plugin extensively. Furthermore, I think the following features would be useful.

After I have highlighted a text on a website or a pdf, I'd like to

- start tts from the beginning (currently doesn't work in a pdf)
- pause tts
- restart from the beginning (even it if is not highlighted anymore, but was started before)
- jump back sentences (with faster speed sometimes you need to listen to certain parts twice; jumping forward does not really make sense imo)

On top of that, I think that it is important to keep newlines between titles and paragraphs such that the tts engine produces a short pause between them. Sometimes they probably have to be added manually.

Finally, I'd like to say that live sentence highlighting sounds very hard to implement well and it is not really a necessary feature. 

I think this could easily become the best tts extension for chrome. Great work so far.

**Comment by pgmichael** [comment]
Hi there,
Thanks for the feedback!

@ischlag I've opened some seperate issues about your concerns. This should make the development easier to follow.

In addition to this issue, you can check out #16, #17.

**Comment by superluig164** [comment]
At the present moment, since Chrome added the media control button to the top right, you can use that to pause the current selection.  Although it is clunky, it's a good workaround for now, until @pgmichael becomes active on this project again.
